model/event/event.go

```diff
@@ -37,9 +37,9 @@ func (e *EventLogEntry) Processed() (bool, time.Time) {
 }
 
 type unmarshalEventLogEntry struct {
-	ID           bson.ObjectId `bson:"_id" json:"-"`
-	ResourceType string        `bson:"r_type,omitempty" json:"resource_type,omitempty"`
-	ProcessedAt  time.Time     `bson:"processed_at" json:"processed_at"`
+	ID           interface{} `bson:"_id" json:"-"`
```

Comment:
do we actually need to know the id of this type?

Reply:
Yep, we mark events as processed.